### DISCUSSION TITLE
Can Wflow deal with bifurcating channels?

### DISCUSSION THREAD

**Discussion by tnlim** [Deltares/Wflow.jl#319] [Q&A] [answered]
Hi, does Wflow deal with bifurcating channels? According to [this page](https://wflow.readthedocs.io/en/latest/wflow_sbm_old.html) on the older version of Wflow (that runs on PCraster), there is the option to deal with bifurcations. May I know how can I do this using the CLI (that runs on Julia)?

**Comment by JoostBuitink** [comment, marked as answer]
This is sadly not possible in the current version of Wflow, we only allow for confluences and sadly not for bifurcations.
